test/test_operations.py

```diff
@@ -1861,6 +1861,20 @@ def test_emb_bf16(self):
     emb_out = emb(index)
     assert emb_out.dtype == torch.bfloat16
 
+  def test_transpose_1d(self):
+
+    def test_fn(t1):
+      return t1.t()
```

Comment:
Is it true that the non-in-place operation will create a copy of tensor and transpose it?

Reply:
yes